openpype/settings/entities/schemas/projects_schema/schema_project_deadline.json

```diff
@@ -195,6 +195,36 @@
                                 ]
 
                             }
+                        },
+                        {
+                            "type": "list",
+                            "collapsible": true,
+                            "key": "disable_strict_check_profiles",
+                            "label": "Disable Strict Error Check profiles",
+                            "use_label_wrap": true,
+                            "docstring": "Set profile for disabling 'Strict Error Checking'",
+                            "object_type": {
+                                "type": "dict",
```

Comment:
I would add one boolean input which would tell if it should be strict or not. There is no way how to tell: By default enable strict error check, except one very specific case.

Reply:
Or maybe for all `Scene` task types make it strict, but if task name is `TestScene` then ignore it.

Reply:
Maybe: In that case the logic should be reversed to "enable" strict check profiles, so the profile would enable strict mode or disable strict mode?

Reply:
Default is Strict check enabled, that is DL default, even when we didn't sending anything.
This functionality is only as a workaround to push trough some critical publish which is failing on minor recoverable error.

We are now discussing with @m-u-r-p-h-y  if profiles are necessary, if simple boolean toggle wouldn't be enough.